src/platform.ts

```diff
@@ -0,0 +1,35 @@
+import {path, homeDir, fs} from "./adapter.node";
+
+export const isWindows = process.platform === "win32";
+
+let _configDir: () => string;
+
+if (process.platform === "darwin") {
+  _configDir = () => {
+    return path.join(homeDir(), "Library", "Application Support", "edgedb");
+  };
+} else if (process.platform === "win32") {
+  _configDir = () => {
+    const localAppDataDir =
+      process.env.LOCALAPPDATA ?? path.join(homeDir(), "AppData", "Local");
+
+    return path.join(localAppDataDir, "EdgeDB", "config");
+  };
+} else {
+  _configDir = () => {
+    let xdgConfigDir = process.env.XDG_CONFIG_HOME;
+    if (!xdgConfigDir || !path.isAbsolute(xdgConfigDir)) {
+      xdgConfigDir = path.join(homeDir(), ".config");
+    }
+
+    return path.join(xdgConfigDir, "edgedb");
+  };
+}
+
+export function getConfigDir(): string {
+  let configDir = _configDir();
+  if (!fs.existsSync(configDir)) {
+    configDir = path.join(homeDir(), ".edgedb");
+  }
```

Comment:
I'm sorry about the delay - but there's an update in the Python binding to address comments: https://github.com/edgedb/edgedb-python/compare/aeb84cd...a1c817d?expand=1#diff-7390f121f7b0fa5c6846ff46ad4adfd547e45c0bef1dbf194a46873402a779cc

Reply:
Ok, updated